packages/core/src/utils/checkFormInputValue.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Checks if the value passed to FormInput is different than
+ * null, undefined or empty strings.
+ *
+ * Returns an boolean related to whether
```

Comment:
JSDoc has a marker for returns.

Add after `@params`:

```suggestion
 * @return {boolean} whether the field is empty or not
```